exporter/datadogexporter/metadata/gcp/gcp.go

```diff
@@ -0,0 +1,63 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package gcp
+
+import (
+	"fmt"
+
+	"go.opentelemetry.io/collector/consumer/pdata"
+	"go.opentelemetry.io/collector/translator/conventions"
+)
+
+type HostInfo struct {
+	HostAliases []string
+	GCPTags     []string
+}
+
+// HostnameFromAttributes gets a valid hostname from labels
+// if available
+func HostnameFromAttributes(attrs pdata.AttributeMap) (string, bool) {
+	if hostName, ok := attrs.Get(conventions.AttributeHostName); ok {
+		return hostName.StringVal(), true
+	}
+
+	return "", false
```

Comment:
This Codecov warning can be eliminated by adding a test where we call `HostnameFromAttributes` with an empty attribute map, and check that the return is `"", false`.